index.html

```diff
@@ -507,22 +507,42 @@ <h3 id="bp-producers">Guidance for [=Producers=]</h3>
 <section id="guidance-for-consumers">
 <h3 id="bp-consumers">Guidance for [=Consumers=]</h3>
 
+<p>The purpose of collecting, serializing, and transmitting language and [=string direction=] metadata is so that [=consumers=] can use it to display and process string data correctly.</p>
+
 <div class="req" id="bp-consumer-lang-assumption">
-<p class="advisement">Consumers SHOULD consume language metadata provided by document formats or protocols whenever possible. When language metadata is not available, consumers MAY use heuristics to detect the language.</p>
+<p class="advisement">Consumers SHOULD employ any language metadata provided by document formats or protocols when processing or displaying the associated string value.</p>
 </div>
 
 <div class="req" id="bp-consumer-dir-assumption">
-<p class="advisement">Consumers SHOULD consume direction metadata provided by document formats or protocols whenever possible. When direction metadata is not available, consumers SHOULD use first-strong detection (possibly by using the value <code>auto</code> for the direction).</p>
+<p class="advisement">Consumers SHOULD employ any [=string direction=] metadata provided by document formats or protocols when processing or displaying the associated string value.</p>
 </div>
 
 <div class="req" id="bp-consumer-isolation">
-<p class="advisement">When inserting content into a document, and particularly when inserting that content into existing text, consumers SHOULD apply bidirectional isolation via markup or Unicode isolating characters. Such isolation SHOULD use the direction found in document format or protocol supplied metadata if possible.</p>
+<p class="advisement">When displaying a string or inserting a string into a document, consumers SHOULD apply bidirectional isolation to the string value using the [=string direction=] found in document format or protocol metadata when available or first-strong isolation when it is not.</p>
+</div>
+
+<p>To prevent [=spillover effects=] and get the best results, it is always safe to wrap an inserted string value with bidirectional isolation.</p>
```

Comment:
Suggestion:

```
<p>It never causes a problem to wrap an inserted string value with bidirectional isolation, but it prevents [=spillover effects=] and produces the best results.</p>
```

Reply:
@r12a suggested

```suggestion
<p>It never causes a problem to wrap an inserted string value with bidirectional isolation, but it prevents [=spillover effects=] and produces the best results.</p>
```

Reply:
Here's my light edit:

```suggestion
<p>It never causes a problem to wrap an inserted string value with bidirectional isolation, and doing so prevents [=spillover effects=] to produce the best result.</p>
```